docs/background/plot_bgcube_true_reco.py

```diff
@@ -0,0 +1,204 @@
+
+    Parameters
+    ----------
+    input_file1, input_file2 : str
+        File where the corresponding bg cube model is stored.
+    name1, name2 : str
+        Name to use for plot labels/legends.
+    """
+    # get cubes
+    filename1 = input_file1
+    filename2 = input_file2
+    print('filename1', filename1)
+    print('filename2', filename2)
+    bg_cube_model1 = CubeBackgroundModel.read(filename1,
+                                              format='table').background_cube
+    bg_cube_model2 = CubeBackgroundModel.read(filename2,
+                                              format='table').background_cube
+
+    # normalize 1 w.r.t. 2 (i.e. true w.r.t. reco)
+    # normalize w.r.t. cube integral
+    integral1 = bg_cube_model1.integral
+    integral2 = bg_cube_model2.integral
+    bg_cube_model1.data *= integral2/integral1
+
+    # make sure that both cubes use the same units for the plots
+    bg_cube_model2.data = bg_cube_model2.data.to(bg_cube_model1.data.unit)
+
+    # plot
+    fig, axes = plt.subplots(nrows=2, ncols=3)
+    fig.set_size_inches(30., 15., forward=True)
+    group_info = 'group 27: ALT = [72.0, 90.0) deg, AZ = [90.0, 270.0) deg'
+    plt.suptitle(group_info)
+
+    # plot images
+    #  rows: similar energy bin
+    #  cols: same file
+    bg_cube_model1.plot_image(energy=Quantity(1., 'TeV'), ax=axes[0, 0])
+    axes[0, 0].set_title("{0}: {1}".format(name1, axes[0, 0].get_title()))
+    bg_cube_model1.plot_image(energy=Quantity(10., 'TeV'), ax=axes[1, 0])
+    axes[1, 0].set_title("{0}: {1}".format(name1, axes[1, 0].get_title()))
+    bg_cube_model2.plot_image(energy=Quantity(1., 'TeV'), ax=axes[0, 1])
+    axes[0, 1].set_title("{0}: {1}".format(name2, axes[0, 1].get_title()))
+    bg_cube_model2.plot_image(energy=Quantity(10., 'TeV'), ax=axes[1, 1])
+    axes[1, 1].set_title("{0}: {1}".format(name2, axes[1, 1].get_title()))
+
+    # plot spectra
+    #  rows: similar det bin
+    #  cols: compare both files
+    bg_cube_model1.plot_spectrum(coord=Angle([0., 0.], 'degree'),
+                                 ax=axes[0, 2],
+                                 style_kwargs=dict(color='blue',
+                                                   label=name1))
+    spec_title1 = axes[0, 2].get_title()
+    bg_cube_model2.plot_spectrum(coord=Angle([0., 0.], 'degree'),
+                                 ax=axes[0, 2],
+                                 style_kwargs=dict(color='red',
+                                                   label=name2))
+    spec_title2 = axes[0, 2].get_title()
+    if spec_title1 != spec_title2:
+        s_error = "Expected same det binning, but got "
+        s_error += "\"{0}\" and \"{1}\"".format(spec_title1, spec_title2)
+        raise ValueError(s_error)
+    else:
+        axes[0, 2].set_title(spec_title1)
+
+    # plot normalized models on top
+
+    E_0 = E_REF
+    norm = NORM
+    index = INDEX
+
+    plot_data_x = axes[0, 2].get_lines()[0].get_xydata()[:,0]
```

Comment:
From a quick look, it seems this code block is repeated four times with just minor variations?
Can you split it out in a utility function that you call four times?